va/va.c

```diff
@@ -1640,6 +1640,25 @@ VAStatus vaSyncSurface (
   return va_status;
 }
 
+VAStatus vaSyncSurface2 (
+    VADisplay dpy,
+    VASurfaceID surface,
+    uint64_t timeout_ns
+)
+{
+  VAStatus va_status;
+  VADriverContextP ctx;
+
+  CHECK_DISPLAY(dpy);
+  ctx = CTX(dpy);
+
+  va_status = ctx->vtable->vaSyncSurface2( ctx, surface, timeout_ns );
```

Comment:
Need check ctx->vtable->vaSyncSurface2 against NULL firstly. BTW could you delete the whitespace between brace and parameters to keep the same code style in this file?

Reply:
Thanks! Done.

As for the whitespaces, they're present in all other functions, e.g.:
`  va_status = ctx->vtable->vaSyncSurface( ctx, render_target );`
`  va_status = ctx->vtable->vaQuerySurfaceStatus( ctx, render_target, status );`
`  va_status = ctx->vtable->vaQuerySurfaceError( ctx, surface, error_status, error_info );`